Woof/Woof/Model/Sitter/Sitter.swift

```diff
@@ -20,7 +20,7 @@ struct Sitter: Codable, Identifiable {
     /// The URL of the sitter's avatar image.
     var avatarUrl: URL?
 
-    /// The additional information about the sitter, like his experience with dogs, favorite places for walks,
+    /// The additional information about the sitter, like his experience with dogs, favourite places for walks,
```

Comment:
It's spelled correctly, why do you change this?

Reply:
Thanks for your attention. Sure, it's correct spelling. My automatic spelling corrector advised me that and I changed it without checking. Fixed it.Sorry for this.